dracut/30ignition/coreos-teardown-initramfs-network.sh

```diff
@@ -4,15 +4,30 @@
 
 set -euo pipefail
 
+down_interface() {
+    ip link set $1 down
+    ip addr flush dev $1
+    rm -f -- /tmp/net.$1.did-setup
+}
+
+# We want to take down the bonded interfaces first
+if [ -f "/sys/class/net/bonding_masters" ]; then
+    bonds="$(cat /sys/class/net/bonding_masters)"
+    for b in ${bonds[@]}; do
+        down_interface ${b}
+        echo -"${b}" > /sys/class/net/bonding_masters
```

Comment:
might be worth linking to the documentation about `echo '-bond0' > /sys/class/net/bonding_masters` being the appropriate interface. Or maybe we don't want to since that may incur another testing cycle.